src/Core/TestInterface.php

```diff
@@ -90,6 +90,13 @@ public function getPending();
     public function setPending($state);
 
     /**
+     * Return whether or not the test is focused
+     *
+     * @return bool
+     */
+    public function getFocused();
```

Comment:
Yeah, that makes some degree of sense. Since the focused state doesn't use `null`, it can probably be `isFocused()` then. I'll update that in a sec.